docs/events/api/timeline-api-get.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[timeline-api-get]]
+=== Get timelines or timeline templates
+
+Get timelines or timeline templates.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/timelines`
+
+==== Request params
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`only_user_favorite` |Boolean a|Returns only favorite timelines / timeline templates if set to `true`. Default to `false`
+|No
+|`page_index` |Number |Page number of the result
+|No
+|`page_size` |Number |Items per page in result
+|No
+|`search` |String |The key word of the timelines / timeline templates you are searching for
+|No
+|`sort_field` |String |The order you want the result to be arranged. Can be `title`, `description`, `updated`, `created`
+|No
+|`sort_order` |String |`asc` or `desc`
+|No
+|`status` |String |Can be `active` or `immutable`. `active` returns all the custom timelines / timeline templates, whereas `immutable` returns Elastic prebuilt templates.
+|No
+|`timeline_type` |String |Use the `timeline` object's <<timeline-object-typeField, `timelineType`>> field
+to determine whether a timeline or a timeline template is created, where:
+
+`default` returns timelines, `template` returns timeline templates
+|No
+
+|==============================================
+
+===== Example requests
+
+Get timelines:
+
+[source,console]
+--------------------------------------------------
+GET api/timelines?page_size=10&page_index=1&sort_field=updated&sort_order=desc&timeline_type=default
+
+--------------------------------------------------
+
+
+=== Get timeline / timeline template by savedObjectId
+
+Get single timeline or timeline template by savedObjectId.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/timeline?id=<savedObjectId>`
+
+===== Example requests
+
+Get timeline by savedObjectId:
+
+[source,console]
+--------------------------------------------------
+GET /api/timeline?id=<savedObjectId>
+
+--------------------------------------------------
+
+
+=== Get timeline template by templateTimelineId
+
+Get single timeline template by <<timeline-schema, templateTimelineId>>.
```

Comment:
Hi @angorayc - the `<<timeline-schema>>` reference is invalid, hence the error. What reference were you trying to link to here? I do see a `timeline-object-schema` reference elsewhere - not sure if that's what you wanted?

Reply:
timeline-object-schema it is.